### PR TITLE
Add indonesian translations

### PR DESCRIPTION
**Pull request by utsmannn**
- Add indonesian translation
- Update gradle version to 4.0.1
- Update kotlin version to 1.3.72

### CODE REVIEW

File: blitz/src/main/res/values-in/strings.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<resources>
+    <string name="blitz_future">nanti</string>
+    <string name="blitz_now">baru saja</string>
+    <string name="blitz_single_minute">semenit yang lalu</string>
+    <string name="blitz_single_hour">sejam yang lalu</string>
+    <string name="blitz_single_day">kemarin</string>
+    <string name="blitz_single_week">seminggu yang lalu</string>
+    <string name="blitz_single_month">sebulan yang lalu</string>
+    <string name="blitz_single_year">setahun yang lalu</string>
+</resources>
```
Comment (Perfomer) [inline blitz/src/main/res/values-in/strings.xml:11]: What about plurals?

Comment (utsmannn) [reply blitz/src/main/res/values-in/strings.xml:11]: there is no plural in the adverb of time in the indonesian language

Comment (utsmannn) [reply blitz/src/main/res/values-in/strings.xml:11]: Sorry I forget, thanks dude

**Comment by Perfomer** [comment]
Thank you!

**Pull request merged**